doc/content/source/bcs/SakiyamaElectronDiffusionBC.md

```diff
@@ -1,20 +1,31 @@
 # SakiyamaElectronDiffusionBC
 
-!alert construction title=Undocumented Class
-The SakiyamaElectronDiffusionBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/SakiyamaElectronDiffusionBC
 
 ## Overview
 
-!! Replace these lines with information regarding the SakiyamaElectronDiffusionBC object.
+`SakiyamaElectronDiffusionBC` is a thermal outflow boundary condition.
+
+The thermal driven outflow is defined as
+
+\begin{equation}
+\Gamma_{e} \cdot \textbf{n} = \frac{1}{4}\sqrt{\frac{8e}{\pi m_{e}} \frac{2}{3} \frac{n_{\varepsilon}}{n_{e}}}n_{e}
+\end{equation}
```

Comment:
@gsgall I would recommend not doing this edit because as originally written the formulation of thermal flux is clearer (i.e. $\Gamma \cdot \textbf{n} = v_{th} n_{e}$), in my opinion. Please let me know if you have strong feelings about this edit.

Reply:
That's good with me. No strong feelings here.

Reply:
We can mark this and all the others with a similar comment resolved.